toko/apps/products/views.py

```diff
@@ -1,18 +1,54 @@
-from django.shortcuts import render
+from operator import __or__ as OR
+
+from django.db.models import Q
 from django.views.generic.detail import DetailView
 from django.views.generic.list import ListView
-from .models import Product
+
+from .models import Product, Category
+
+
+class CategoryDetailView(DetailView):
+    """Return category detail."""
+
+    template_name = 'categories/detail.html'
+    model = Category
+    context_object_name = 'object_category'
```

Comment:
Nah, itulah kenapa tadi saya saranin related_name untuk field `categories` diubah menjadi `products`. Jadi nanti disini manggilnya `category.products.all` soalnya yang di loop disini memang list of products, bukan categories.